lib/utils.pm

```diff
@@ -530,6 +531,19 @@ sub sle_version_at_least {
     die "unsupported SLE $version_variable $version in check";
 }
 
+=head2 sle_version_convert
+
+    sle_version_convert([$version]);
+
+Helper for parsing SLE RC version into integer. It replaces SLE version
+in format X-SPY into X.Y.
+=cut
+sub sle_version_convert {
```

Comment:
hm, the name "sle_version_convert" is a bit generic, how about `scc_version` ?

Reply:
@okurz Good idea, thanks. Changed.